### PR TITLE
Use matrix-appservice-bridge logging

### PR DESCRIPTION
**Pull request by Half-Shot**
Counterpart to https://github.com/matrix-org/matrix-appservice-bridge/pull/412

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "@types/ajv": "^1.0.0",
     "@types/chai": "^4.2.22",
     "@types/cors": "^2.8.12",
-    "@types/express": "^4.17.13",
+    "@types/express": "4.17.13",
```
Comment (jaller94) [inline package.json:83]: What's so glorious about this particular version? 🤔

File: src/figma/index.ts [hunk 2]
```diff
@@ -15,8 +17,8 @@ interface FigmaWebhookDefinition {
     description: string;
 }
 
-const log = new LogWrapper('FigmaWebhooks');
-
+const log = new Logger('FigmaWebhooks');
+
```
Comment (jaller94) [inline src/figma/index.ts:21]: Space character in an otherwise empty line.
But no harm done.

**Review by jaller94 — APPROVED** [review]
Looks good to me.

File: package.json (hunk 1 above)
Comment (Half-Shot) [reply package.json:83]: I think it was typing conflicts with m-a-b. Tedious. I'll see if I can stop pinning.

**Pull request merged**